### PR TITLE
finish exercise 5

### PR DESCRIPTION
**Pull request by talivneh**
Hi Mentors,
Sorry for the delay.

Thank you :)

### CODE REVIEW

File: src/solution/server/package.json [hunk 1]
```diff
@@ -16,6 +16,7 @@
     "body-parser": "^1.20.0",
     "dotenv": "^16.0.1",
     "express": "^4.18.1",
+    "mysql2": "^2.3.3",
```
Comment (liorshamian) [inline src/solution/server/package.json:19]: you should have add sequelize package as well

File: src/solution/dist/clients/item_client.js [hunk 2]
```diff
@@ -29,7 +29,7 @@ export default class ItemClient {
     }
   }
 
-  async updateComplete(id, fields) {
+  async updateStatus(id, fields) {
```
Comment (liorshamian) [inline src/solution/dist/clients/item_client.js:32]: you could call to 
 fetch('/item',..) 
(without localhost:8080)

File: src/solution/server/db/config/config.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
+{
```
Comment (liorshamian) [inline src/solution/server/db/config/config.json:1]: you have the config.json file twice - in db & db/config..
should be only once

**Review by liorshamian — COMMENTED** [review]
where is .sequelizerc file?
all in all good job!

**Comment by talivneh** [comment]
Thanks for the feedback!

**Pull request merged**